dhis-2/dhis-services/dhis-service-validation/src/main/java/org/hisp/dhis/outlierdetection/service/DefaultOutlierDetectionService.java

```diff
@@ -212,4 +216,28 @@ private OutlierDetectionMetadata getMetadata( OutlierDetectionRequest request, L
         metadata.setMaxResults( request.getMaxResults() );
         return metadata;
     }
+
+    /**
+     * Returns outlier values using the algorithm defined in the request.
+     *
+     * @param request the {@link OutlierDetectionRequest}.
+     * @return a {@link OutlierDetectionMetadata} instance.
+     */
+    private List<OutlierValue> getOutliers( OutlierDetectionRequest request )
+    {
+        if ( request.getAlgorithm() == OutlierDetectionAlgorithm.Z_SCORE )
+        {
+            return zScoreOutlierDetectionManager.getOutlierValues( request );
```

Comment:
Not related to the actual task but when I look at this names like `zScoreOutlierDetectionManager` and `minMaxOutlierDetectionManager` don't make for good reading. And as the classes already have that information I'd think they can have shorter names.

Reply:
Right, thought about it, I have shortened it a little now.